setup.json

```diff
@@ -107,7 +107,8 @@
       "prospector==1.2.0",
       "pylint==2.4.4",
       "toml~=0.10.0",
-      "yapf==0.28.0"
+      "yapf==0.28.0",
+      "mypy==0.780"
```

Comment:
We don't need this, right? Since we install in the `pre-commit` from the repo and this shouldn't rely on the `system`? At least you haven't marked it as such in the `pre-commit-config.yaml`. Or does this depend on being able to import resources from other dependencies in the environment?

Reply:
Yup, you're right. I'm just used to also putting it here so that my editor picks up the right version - but that sounds like a me problem 😄